src/App.tsx

```diff
@@ -15,48 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState<SortType>(SortType.NONE);
+  const [isReversed, setReversed] = useState<boolean>(false);
```

Comment:
Let's use naming convention for the useState `const [value, setValue] = useState()`:
`const [ownerName, setOwnerName] = useState('Ihor')`
```suggestion
  const [isReversed, setIsReversed] = useState<boolean>(false);
```